src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,71 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
+    private static final String DEFAULT_SUPPLY = "supply";
+    private static final String DEFAULT_BUY = "buy";
+    private static final String DEFAULT_RESULT = "result";
+    private static final String COMA = ",";
```

Comment:
```suggestion
    private static final String DELIMITER = ",";
```
could be SEPARATOR too
and it's `comma` just FYI